test/src/test/java/org/corfudb/runtime/collections/UFOStoreTestImpl.java

```diff
@@ -0,0 +1,145 @@
+package org.corfudb.runtime.collections;
+
+import com.google.protobuf.Message;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.CorfuStoreMetadata;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.lang.reflect.InvocationTargetException;
+import java.util.Collection;
+import java.util.List;
+
+/**
+ * UFOStore is a thin layer over CorfuStore that provides certain metadata management
+ * that carry business logic specific to verticals.
+ *
+ * Created by hisundar on 2020-09-16
+ */
+class UFOStoreTestImpl {
```

Comment:
Looks like UFOStoreTestImpl and UFOTxnTestImpl.java are just wrapping the CorfuStore and TxnContext with no other logic, so why do we need them? Is it because you are going to add new logics later?

Reply:
yes the latter, I am still in the process of porting the metadata revision, timestamp logic there.